css-color-4/Overview.bs

```diff
@@ -4671,16 +4671,25 @@ Interpolating with Alpha</h3>
 Hue Interpolation</h3>
 
 	For color functions with a hue angle (LCH, HSL, HWB etc), there are multiple ways to interpolate.
-	We typically want to avoid arcs over 360 for the difference between the angles, as they are rarely desirable,
-	so in most cases angles are fixed up prior to interpolation so that per-component interpolation is done over less than 360 degrees, often less than 180.
+	As arcs greater than 360&deg; are rarely desirable,
+	by default hue angles are fixed up prior to interpolation
+	so that per-component interpolation is done over less than 360&deg;, often less than 180&deg;.
 
 	Host syntax can specify any of the following algorithms for hue interpolation
 	(angles in the following are in degrees, but the logic is the same regardless of how they are specified).
 	Specifying a hue interpolation strategy is already part of the <<color-interpolation-method>> syntax
 	via the <<hue-interpolation-method>> token.
 
-	Unless the type of hue interpolation is ''specified'', both angles need to be constrained to [0, 360) prior to interpolation.
-	One way to do this is <code><i>θ</i> = ((<i>θ</i> % 360) + 360) % 360</code>.
+	Unless the type of hue interpolation is ''specified'',
+	both angles and their difference need to be constrained to [0, 360] prior to interpolation.
+	To do this, the minimum number of turns that fit in the lesser angle
+	is added subtracted from both angles,
+	bringing the lesser angle into the range [0,360);
+	and if the difference between them is greater than 360&deg;
+	then the minimum number of turns to bring the difference into the range [0,360]
+	is further subtracted from the greater angle.
+
+	Note: Differences of 360&deg; are allowed to maintain the originally specified direction of a full turn.
```

Comment:
But once normalized, a difference of 360 is no longer possible. So this has to start with the specified value. See https://github.com/w3c/csswg-drafts/issues/7750